courses/free-courses-en.md

```diff
@@ -307,6 +307,7 @@
 
 ### Blockchain
 
+* [BerkeleyX: Blockchain Technology](https://www.edx.org/learn/blockchain/university-of-california-berkeley-blockchain-technology?index=product&objectID=course-17b2db20-bb6d-4148-ad0b-9b9b99e287c1&webview=false&campaign=Blockchain+Technology&source=edX&product_category=course&placement_url=https%3A%2F%2Fwww.edx.org%2Flearn%2Fblockchain) - Rustie Lin and Nadir Akhtar
```

Comment:
do we know that the url doesn't change after it's archived? are all the url parameters neede? I get hte same page with https://www.edx.org/learn/blockchain/university-of-california-berkeley-blockchain-technology